apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/AbstractR4ResourceProvider.java

```diff
@@ -277,25 +278,39 @@ public T read(@IdParam IdType claimId, RequestDetails requestDetails) {
   @Trace
   public Bundle findByPatient(
       @RequiredParam(name = "mbi")
-          @Description(shortDefinition = "The patient identifier to search for")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PAC_MBI,
+              value = OpenAPIContentProvider.PAC_MBI_VALUE)
           ReferenceParam mbi,
       @OptionalParam(name = "type")
-          @Description(shortDefinition = "A list of claim types to include")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_CLAIM_TYPE_SHORT,
+              value = OpenAPIContentProvider.EOB_CLAIM_TYPE_VALUE)
           TokenAndListParam types,
       @OptionalParam(name = "startIndex")
-          @Description(shortDefinition = "The offset used for result pagination")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PATIENT_START_INDEX_SHORT,
+              value = OpenAPIContentProvider.PATIENT_START_INDEX_VALUE)
           String startIndex,
       @OptionalParam(name = "isHashed")
-          @Description(shortDefinition = "A boolean indicating whether or not the MBI is hashed")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.PAC_IS_HASHED,
+              value = OpenAPIContentProvider.PAC_IS_HASHED_VALUE)
           String hashed,
       @OptionalParam(name = "excludeSAMHSA")
-          @Description(shortDefinition = "If true, exclude all SAMHSA-related resources")
+          @Description(
+              shortDefinition = OpenAPIContentProvider.EOB_EXCLUDE_SAMSHA_SHORT,
+              value = OpenAPIContentProvider.EOB_EXCLUDE_SAMSHA_VALUE)
```

Comment:
There's one extra element to this with PAC data in that `ClaimResponse` doesn't contain any SAMHSA potential data, but we still exclude any `ClaimResponse` resource when the associated `Claim` resource contains SAMHSA data.  Reason being that `Claim` and `ClaimResponse` are a 1 to 1 relationship and serving a `ClaimResponse` but excluding it's associated `Claim` seemed undesired.